ipld/resolver/src/service.rs

```diff
@@ -301,6 +296,7 @@ where
             BehaviourEvent::Discovery(e) => self.handle_discovery_event(e),
             BehaviourEvent::Membership(e) => self.handle_membership_event(e),
             BehaviourEvent::Content(e) => self.handle_content_event(e),
+            BehaviourEvent::ConnectionLimits(_) => {}
```

Comment:
any log you want to emit here?

Reply:
The argument is `type Void`, so carries zero information. The `ConnectionHandler` is a `dummy`. So the path should never be reached, however, I wasn't comfortable putting an `unreachable!` there.